examples/iscsi-pd/README.md

```diff
@@ -0,0 +1,56 @@
+# How to Use it?
+Here is my setup to setup Kubernetes with iSCSI persistent storage. I use Fedora 21 on Kubernetes node. 
+
+Install iSCSI initiator on the node:
+
+    # yum -y install iscsi-initiator-utils
+   
+   
+then edit */etc/iscsi/initiatorname.iscsi* and */etc/iscsi/iscsid.conf* to match your iSCSI target configuration.
+
+I mostly follow these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi&f=2) to setup iSCSI initiator and these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi) to setup iSCSI target.
+
```

Comment:
The README will live in examples/iscsi-pd and be the canonical guide for mounting iSCSCI disks.